Controllers/BeersControllers.cs

```diff
@@ -0,0 +1,89 @@
+using Microsoft.AspNetCore.Mvc;
+using BeerAPI.Models; 
+using BeerAPI.Services; 
+
+namespace BeerAPI.Controllers
+{
+    [ApiController]  
+    [Route("api/[controller]")] 
+
+    //? Step 2: DATA STORE
+    public class BeersController : ControllerBase 
+    {
+        private static List<Beer> beers = new List<Beer>  
+        {
+            new Beer { Id = 1, Name = "Lager", Price = 5.99M },
+            new Beer { Id = 2, Name = "IPA", Price = 6.49M, PromoPrice = 3.99M }
+        };
+
+        private object _beerDescriptionService;
+
+        //! INJECTION
+        public BeersController(IBeerDescriptionService beerDescriptionService)
+        {
+            _beerDescriptionService = beerDescriptionService; 
+        }
+
+        // GET REQUEST: return of info given a BeerId
+        [HttpGet("{id}")] 
+        public ActionResult<Beer> GetBeerById(int id) 
```

Comment:
Better to return `IActionResult` to since error scenarios generally return a result that won't have a `Beer` object.
```suggestion
        public IActionResult GetBeerById(int id) 
```